tests/src/FunctionalJavascript/ArticleCreationTest.php

```diff
@@ -40,26 +40,26 @@ public function testCreateArticle() {
     // Add Text Paragraph.
     $this->addTextParagraph(static::$paragraphsField, 'Awesome text');
 
-    // Add Gallery Paragraph.
+    // Add Gallery Paragraph between Image and Text.
     $this->addGalleryParagraph(static::$paragraphsField, 'Test gallery', [
       'media:1',
       'media:5',
-    ]);
+    ], 1);
 
     // Add Quote Paragraph.
     $this->addTextParagraph(static::$paragraphsField, 'Awesome quote', 'quote');
 
-    // Add Twitter Paragraph.
-    $this->addSocialParagraph(static::$paragraphsField, 'https://twitter.com/ThunderCoreTeam/status/776417570756976640', 'twitter');
+    // Add Twitter Paragraph between Text and Quote.
+    $this->addSocialParagraph(static::$paragraphsField, 'https://twitter.com/ThunderCoreTeam/status/776417570756976640', 'twitter', 3);
 
     // Add Instagram Paragraph.
     $this->addSocialParagraph(static::$paragraphsField, 'https://www.instagram.com/p/BK3VVUtAuJ3/', 'instagram');
 
     // Add Link Paragraph.
     $this->addLinkParagraph(static::$paragraphsField, 'Link to Thunder', 'http://www.thunder.org');
 
-    // Add Video paragraph.
-    $this->addVideoParagraph(static::$paragraphsField, ['media:7']);
+    // Add Video paragraph at the beginning.
+    $this->addVideoParagraph(static::$paragraphsField, ['media:7'], 0);
```

Comment:
Maybe we should consider to improve checks (assertions), that also order is taken in account. fe. we are checking that "Awesome text" exists on page, but we are not checking that it's 4th paragraph.